platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/style/layers/TransitionOptions.java

```diff
@@ -35,10 +48,27 @@ public TransitionOptions(long duration, long delay) {
    * @return a new transition property object
    */
   @Keep
+  @Deprecated
```

Comment:
missing deprecated javadoc indicating alternative method to call

Reply:
It's an internal API, but won't hurt to include an entry I guess :+1: